stride/problem/geometry.py

```diff
@@ -283,6 +284,46 @@ def default(self, geometry_type, *args, **kwargs):
 
             self.add(index, self._transducers.get(0), _coordinates)
 
+    def from_fullwave(self, geom_path, **kwargs):
+        """
+        Populates geometry container based from a Fullwave geometry pgy file
+
+        Parameters
+        ----------
+        geom_path : str
+            Path to .pgy Fullwave geometry file
+        scale: float, optional
+            Value to each scale the location values in all dimensions. Useful for unit conversion.
+            To transform cell units from Fullwave to metric units in Stride, scale should be the
+            same value as the model grid-spacing. Default 1.
+        disp: tuple or float, optional
+            Amount to displace in each dimension. Applied after scale. Default (0., 0., 0.)
```

Comment:
What are the units of this displacement? Metres I presume?

Reply:
See commit [4561726](https://github.com/trustimaging/stride/pull/30/commits/4561726a69d476df160dfcfe4964b51b81859c37)